projects/driver_controls/inc/stm32f0xx/ads1015_def.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#define ADS1015_I2C_ADDRESS_GND ((uint8_t)0x48)
```

Comment:
In this case, I think it would make more sense to define this as a base address or similar since your addresses are offset from this.